rifle.py

```diff
@@ -87,6 +94,26 @@ def draw_terminal(self, screen):
     def new_word(self):
         return self.word_generator.request_word(3,3)
 
+class RifleProjectile(): 
+
+	def __init__(self, universe, current_text):
+		self.universe = universe 
+		self.input_ = current_text
+
+	def update(self,events): 
+		target_ID = getTargetID(input_)
+		if (targetID): 
+			target_position = self.universe.enemies_[target_ID].position_ 	
+			print(target_position)	#test 
+			# shoot projectile towards where enemy was at that time. 
+
+	def getTargetID(input): 
+		for enemy in self.universe_.enemies(): 
+			target_tag_word = self.target_tags_[enemy.ID_]
+			if(input == target_tag_word)
```

Comment:
Also, if statement in python don't need bracket, remove them for better python code :)